input/core/src/commonMain/kotlin/majestic/editor/ToolBarHost.kt

```diff
@@ -77,15 +85,10 @@ fun ToolBarHost(
                     modifier = Modifier
                         .wrapContentSize()
                         .clickable(onClick = { controller.select(toolBar) })
-                        .padding(top = 8.dp)
                         .hoverable(interactionSource = interactionSource)
                         .borderBottom(
-                            width = 2.dp,
-                            color = when {
-                                active -> colors.underline.active
-                                hovered -> colors.underline.inActive.copy(alpha = 178F)
-                                else -> Color.Transparent
-                            }
+                            color = if (active) activeColor else if (hovered) inActiveColor.copy(alpha = 178F) else Color.Transparent,
+                            width = 2.dp
```

Comment:
Switch this to a when statement. It makes the code more readable. Use this as a general rule when you are already creating more branches (more than 2) with your if statements